english/blackAndwhite/blackAndwhite.md

```diff
@@ -16,4 +16,11 @@ wearing a leather hood and jeans, smirks seeing Christy tremble in fear.
 Christy sprints to the window to check if someone is hiding there and sees no one. Stranger drifts into the eternal dark
 and leaves a puzzle behind for her.
 
-Christy's phone vibrates and drops off the table and Christy screams in dismay waking up the entire dorm.
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.